definitions/infra-prometheus-redis/definition.yml

```diff
@@ -0,0 +1,16 @@
+domain: INFRA
+type: REDIS
+identifier: targetName
```

Comment:
I'm not too sure about this. Any ideas? I don't get a lot of dimensions when using the POMI integration.

Reply:
Mmmm it's a good question.. We're not redis experts so it will be hard for someone on our side to say anything definitive here :( 

Could you perhaps clarify with the customer you're doing the definition for? Or somehow gather more data to verify?

We currently do not have any test process or pattern of support of any shape for this - I'm totally open to suggestions on how we can improve this!

Reply:
I'm getting this data through POMI which only gives me the cluster_name. The redis exporter itself also doesn't expose a hostname or a different name. I'm conflicted on this. 

In this case I feel the customer should add a custom label to it's data to get the right name, but targetName is definitely a good default. Is there a way to say, take this first, then this. 

In this case that could be: custom label (hostname), and as fallback targetName.